### PR TITLE
Update Metric-template.md 

### PR DESCRIPTION
**Pull request by peculiaruc**
Updated the Metric template to use the New Metrics Template.
- Removed the following sections: 
    1. Description
    2. Objective
    3. Tool proving metric
 
- Moved the tags to the bottom of the metrics template
- Added the following section:
    1. Overview 
    2. Want to know more?- as a dropdown that contains the Data Collection Strategies, Filters and Visualization.

### CODE REVIEW

**Comment by GeorgLink** [comment]
Looks good. Can you please add empty lines between headings and the next text? Some markdown interpreters require the empty lines or else think the heading continues.

**Comment by peculiaruc** [comment]
> Looks good. Can you please add empty lines between headings and the next text? Some markdown interpreters require the empty lines or else think the heading continues.

I have added an empty line between the headings and the next text.

File: community-templates/metric-template.md [hunk 1]
```diff
@@ -1,40 +1,51 @@
...
+*   How does measuring this metric help inform health and sustainability?
+*   If applicable, how does this metric inform DEI-related issues?
 
-- A community manager is interested in this metric because...
-- An OSPO manager is interested in this metric because...
-- An event organizer is interested in this metric because...  
+## Want to Know More?
 
-Each metric has the potential to influence diversity, equity, and inclusion on an open source project. Please consider these factors when framing metrics objectives. 
+<span markdown="1"><details>
```
Comment (ElizabethN) [inline community-templates/metric-template.md:17]: Let's remove this line as I think the <span> and <summary> lines needs to be together

**Comment by peculiaruc** [comment]
@ElizabethN  The line has been removed

**Pull request merged**